### PR TITLE
Today's dogfooding exercise: fix issues #70 and #79 + minor tweaks

### PR DESCRIPTION
**Pull request by simontoens**

### CODE REVIEW

File: plugin-libs/plugin-command/src/main/java/com/salesforce/bazel/eclipse/command/BazelWorkspaceCommandRunner.java [hunk 1]
```diff
@@ -264,7 +264,7 @@ public File computeBazelWorkspaceOutputBase() {
         if (bazelOutputBaseDirectory == null) {
             try {
                 ImmutableList.Builder<String> argBuilder = ImmutableList.builder();
-                argBuilder.add("info").add("execution_root");
+                argBuilder.add("info").add("output_base");
```
Comment (simontoens) [inline plugin-libs/plugin-command/src/main/java/com/salesforce/bazel/eclipse/command/BazelWorkspaceCommandRunner.java]: This fixed the particular "jar not found" issue I was seeing.  I have used this change all day without issues, and it also matches the method name ... (?)

File: plugin-core/src/main/java/com/salesforce/bazel/eclipse/projectview/ProjectViewEditor.java [hunk 2]
```diff
@@ -104,6 +104,12 @@ protected void editorSaved() {
         List<BazelPackageLocation> packageLocations = new ArrayList<>(projects.length);
         for (IJavaProject project : projects) {
             // get the target to get at the package path
+            List<String> targets = BazelEclipseProjectSupport.getBazelTargetsForEclipseProject(project.getProject(), false);
+            if (targets == null || targets.isEmpty()) {
+                // this shouldn't happen, but if it does, we do not want to blow up here
+                // instead return null to force re-import
+                return null;
+            }
```
Comment (simontoens) [inline plugin-core/src/main/java/com/salesforce/bazel/eclipse/projectview/ProjectViewEditor.java]: I ran into a situation where the inner Eclipse was in a bad state, and targets was null...

File: plugin-core/src/main/java/com/salesforce/bazel/eclipse/launch/BazelLaunchConfigurationTab.java [hunk 3]
```diff
@@ -313,12 +313,13 @@ public String getText(Object element) {
...
             BazelLabel label = ((BazelLabel)element);
             TargetKind targetKind = BazelLaunchConfigurationTab.this.lookupLabelKind(label);
-            return label.getLabel() + " (" + targetKind.getKind() + ")";
+            return toFriendlyLabelRepresentation(label, targetKind);
         }
     };
+
+    private static String toFriendlyLabelRepresentation(BazelLabel label, TargetKind targetKind) {
+        return label.getLastComponentOfTargetName() + " (" + targetKind.getKind() + ") - " + label.getLabel();
+    }
```
Comment (simontoens) [inline plugin-core/src/main/java/com/salesforce/bazel/eclipse/launch/BazelLaunchConfigurationTab.java]: When picking a bazel target to launch, make the list of targets easier to read.

File: plugin-core/src/main/java/com/salesforce/bazel/eclipse/launch/BazelJavaSourceLookupDirector.java [hunk 4]
```diff
@@ -33,17 +33,23 @@
  */
 package com.salesforce.bazel.eclipse.launch;
```
Comment (simontoens) [inline plugin-core/src/main/java/com/salesforce/bazel/eclipse/launch/BazelJavaSourceLookupDirector.java]: With these changes setting breakpoints in source code of jars (dependencies) works

File: plugin-libs/plugin-command/src/main/java/com/salesforce/bazel/eclipse/command/BazelWorkspaceCommandRunner.java (hunk 1 above)
Comment (plaird) [reply plugin-libs/plugin-command/src/main/java/com/salesforce/bazel/eclipse/command/BazelWorkspaceCommandRunner.java]: And it was changed recently in PR #75 so matches the timeline of regression.

**Pull request merged**